doc/CodeDocumentation.conf.in

```diff
@@ -810,7 +810,11 @@ INPUT                  = @MFEM_SOURCE_DIR@/doc/CodeDocumentation.dox \
                          @MFEM_SOURCE_DIR@/miniapps/shifted \
                          @MFEM_SOURCE_DIR@/miniapps/solvers \
                          @MFEM_SOURCE_DIR@/miniapps/tools \
-                         @MFEM_SOURCE_DIR@/miniapps/toys
+                         @MFEM_SOURCE_DIR@/miniapps/toys \
+                         @MFEM_SOURCE_DIR@/miniapps/spde \
+                         @MFEM_SOURCE_DIR@/miniapps/dpg \
+                         @MFEM_SOURCE_DIR@/miniapps/dpg \
+                         @MFEM_SOURCE_DIR@/miniapps/dpg/util
```

Comment:
@psocratis, this is causing the following Doxygen failure on my machine:

```
mfem/examples/ex25.cpp:931: warning: no uniquely matching class member found for
  CartesianPML::CartesianPML(mfem::Mesh *mesh_, mfem::Array2D< double > length_)
Possible candidates:
  CartesianPML::CartesianPML(Mesh *mesh_, Array2D< double > length_)' at line 73 of file mfem/examples/ex25.cpp
  CartesianPML::CartesianPML(Mesh *mesh_, Array2D< double > length_)' at line 72 of file mfem/examples/ex25p.cpp
  mfem::CartesianPML::CartesianPML(Mesh *mesh_, const Array2D< double > &length_)' at line 44 of file mfem/miniapps/dpg/util/pml.hpp

mfem/examples/ex25p.cpp:1007: warning: no uniquely matching class member found for
  CartesianPML::CartesianPML(mfem::Mesh *mesh_, mfem::Array2D< double > length_)
Possible candidates:
  CartesianPML::CartesianPML(Mesh *mesh_, Array2D< double > length_)' at line 73 of file mfem/examples/ex25.cpp
  CartesianPML::CartesianPML(Mesh *mesh_, Array2D< double > length_)' at line 72 of file mfem/examples/ex25p.cpp
  mfem::CartesianPML::CartesianPML(Mesh *mesh_, const Array2D< double > &length_)' at line 44 of file mfem/miniapps/dpg/util/pml.hpp
```

Reply:
hmm, I guess it's because the same class exists in ex25?

Reply:
This is resolved now, thanks.